src/Rec/Rec.spec.ts

```diff
@@ -0,0 +1,718 @@
+﻿import { describe, expect, it, vi } from "vitest"
+import { Rec } from "../Rec"
+import type { EqualityComparer } from "../EqualityComparer"
+import { OrderingComparer } from "../OrderingComparer"
+import { pipe } from "../Composition"
+import { Option } from "../Option"
+import { String } from "../String"
+
+const caseInsensitiveEqualityComparer: EqualityComparer<string> = {
+    equals: (s1, s2) => s1.toLowerCase() === s2.toLowerCase(),
+}
+
+const byLengthDescComparer: OrderingComparer<string> = {
+    compare: (s1, s2) =>
+        s1.length > s2.length ? -1 : s2.length > s1.length ? 1 : 0,
+}
+
+const numberDesc = OrderingComparer.reverse(OrderingComparer.Number)
+
+describe("Rec", () => {
+    describe("set", () => {
+        it("sets a new key/value on an empty map", () => {
+            expect(
+                pipe(Rec.empty<string, number>(), Rec.set(["Albus", 12]))
+            ).toStrictEqual({ Albus: 12 })
+        })
+
+        it("sets a new key/value on a non-empty map without the same key using default equality", () => {
```

Comment:
```suggestion
        it("sets a new key/value on a non-empty record without the same key using default equality", () => {
```

(and several more below)

Reply:
https://github.com/fp-toolkit/fp-toolkit/pull/67/commits/09f3f5c565a1e8986b5b05dce808421d2981dacf

Reply:
actually https://github.com/fp-toolkit/fp-toolkit/pull/67/commits/1557de68a0bf110778a5e193c05e7f27e6a1714f